app/models/user.rb

```diff
@@ -105,6 +105,7 @@ def registration_facility_id
 
   def full_teleconsultation_phone_number
     defaulted_teleconsult_number = teleconsultation_phone_number || phone_number
+    teleconsultation_isd_code ||= Rails.application.config.country["sms_country_code"]
```

Comment:
I'd love to make the ISD code required, and prepopulate the form input with the country config. We can aggressively force our data to be high quality here, I think.

Reply:
This is only for users who don't have a teleconsultation number added. They will have the number and isd code both set to `null`. We'll make the code required in the form.

Reply:
Ah great point 👍 Nice.